test/oban/queue/executor_test.exs

```diff
@@ -72,6 +72,22 @@ defmodule Oban.Queue.ExecutorTest do
     end
   end
 
+  describe "new/2" do
+    test "tags are included in the job metadata" do
```

Comment:
This test should be included in `test/integration/telemetry_test.exs` instead. Take a look at the assertion around line 50.